.github/workflows/wheels.yml

```diff
@@ -92,3 +97,23 @@ jobs:
       filter: "*torch2.4.0+cu121*"
       execute: ${{ github.repository == 'facebookresearch/xformers' && github.ref_type == 'tag' }}
 
+  upload_pt_rocm6_0:
+    needs: build
+    uses: ./.github/workflows/wheels_upload_s3.yml
+    with:
+      aws_role: "arn:aws:iam::749337293305:role/pytorch_bot_uploader_role"
+      s3_path: s3://pytorch/whl/rocm6.0/
```

Comment:
One thing I'm not sure about is if I got the bucket path correctly. Here I'm just using the path fragment from download.pytorch.org

Reply:
This should be correct